ondewo/nlu/intent.proto

```diff
@@ -940,32 +940,55 @@ message IntentSorting {
     SortingMode sorting_mode = 2;
 }
 
-message CreateTrainingPhraseRequest {
+message UpdateTrainingPhraseBatchRequest {
+    repeated Intent.TrainingPhrase training_phrases = 1;
+}
+
+message TrainingPhraseStatus {
+    oneof phrase_or_status {
+        Intent.TrainingPhrase training_phrase = 1;
+        string error_message = 2;
+    }
+}
+
+message TrainingPhraseStatusBatch{
+    repeated TrainingPhraseStatus training_phrase_statuses = 1;
+}
+
+message CreateTrainingPhraseBatchRequest {
     // Required. The name of the intent for which to create a training phrase for.
     // Needed since the name of the training phrase may not be set upon creation.
     // Format: `projects/<Project ID>/agent/intents/<Intent ID>`.
-    string intent_name = 1;
+    message CreatePhraseRequest{
+        string intent_name = 1;
 
-    // Required. The training phrase to create.
-    Intent.TrainingPhrase training_phrase = 2;
+        // Required. The training phrase to create.
+        Intent.TrainingPhrase training_phrase = 2;
 
-    // Required. The language of training phrases
-    string language_code = 3;
+        // Required. The language of training phrases
+        string language_code = 3;
+    };
+    repeated CreatePhraseRequest training_phrase_requests = 1;
 }
 
-message GetTrainingPhraseRequest {
+message GetTrainingPhraseBatchRequest {
     // Required. The name of the training phrase.
     // Format: `projects/<Project ID>/agent/intents/<Intent ID>/trainingPhrases/<Training Phrase ID>`
-    string name = 1;
+    repeated string names = 1;
 }
 
-message UpdateTrainingPhraseRequest {
-    // Required. The training phrase to update.
-    Intent.TrainingPhrase training_phrase = 1;
-}
-
-message DeleteTrainingPhraseRequest {
+message DeleteTrainingPhraseBatchRequest {
     // Required. The name of the training phrase.
     // Format: `projects/<Project ID>/agent/intents/<Intent ID>/trainingPhrases/<Training Phrase ID>`
-    string name = 1;
-}
\ No newline at end of file
+    repeated string names = 1;
+}
+
+message DeleteTrainingPhraseStatuses{
```

Comment:
This guy is used as the return message for the endpoint.
Should end in "Response" to maintain consistency with the rest of the API.

You could replace the word "Statuses" with "Response" for example.

Same for the message `TrainingPhraseStatusBatch`

Reply:
renamed into `TrainingPhraseStatusBatchResponse`